options/locale/locale_en-US.ini

```diff
@@ -753,6 +753,7 @@ issues.tracker = Time Tracker
 issues.start_tracking_short = Start
 issues.start_tracking = Start Time Tracking
 issues.start_tracking_history = `started working %s`
+issues.tracker_auto_close = Timer would be stopped automatically when this issue is closed
```

Comment:
Use `Timer will be stopped automatically when this issue gets closed` instead

Reply:
Done @JonasFranzDEV